### PR TITLE
HY-1894 Add BrowserFixes

### PR DESCRIPTION
**Pull request by todbachman-wf**
**Jira Ticket:** [HY-1894](https://jira.webfilings.com/browse/HY-1894)

## Problem
We have used common browser hacks in several other repositories. Ultimately that means we're maintaining the same utilities in multiple places.

## Solution
Consolidate those hacks in common.

## Unit Tests
Migrated BrowserFixesSpec to this repo as well.

## How to QA/+10
1. CI Build Passes

### CODE REVIEW

File: src/BrowserFixes.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            }
+        },
+
+        /**
+         * Unblit is a companion to `applyDisappearingElementFix`, used to force
+         * the browser to remove aliasing after scaling the target.
+         *
+         * @param {HTMLElement} domElement
+         */
+        unblit: function(domElement) {
```
Comment (robbecker-wf) [inline src/BrowserFixes.js]: hmm not sure how I feel about the term blit as that means to copy a bitmap directly into video memory. 
How about forceRasterize or something?

**Comment by robbecker-wf** [comment]
+1 with minor gripe on naming

**Comment by rmconsole-wf** [comment]
When this pull is merged I will not be able to function 
		properly because no valid fixversions exist for 
		Project: HY
		Repo: Workiva/wf-common

***
*Last updated on Wednesday, February 11 05:35 PM CST*

File: src/BrowserFixes.js (hunk 1 above)
Comment (todbachman-wf) [inline src/BrowserFixes.js]: I'm just moving code around here, not looking at the content of it necessarily. ;)

Comment (robbecker-wf) [inline src/BrowserFixes.js]: I noticed this when looking at the other PR. Ok for now.

**Comment by timmccall-wf** [comment]
+10

**Comment by charliekump-wf** [comment]
@rmconsole-wf

**Comment by charliekump-wf** [comment]
QA Resource Approval: +1

- [x] Testing instruction
- [x] Dev +1's
- [x] Dev/QA +10 with detail of what was tested
- [x] Unit test created/updated  
- [x] All unit tests pass 

Merging into master.

**Pull request merged**